support/ruby/collins-shell/collins_shell.gemspec

```diff
@@ -1,95 +1,70 @@
-    "Rakefile",
-    "VERSION",
-    "bin/collins-shell",
-    "collins_shell.gemspec",
-    "lib/collins_shell.rb",
-    "lib/collins_shell/asset.rb",
-    "lib/collins_shell/asset_type.rb",
-    "lib/collins_shell/cli.rb",
-    "lib/collins_shell/console.rb",
-    "lib/collins_shell/console/asset.rb",
-    "lib/collins_shell/console/cache.rb",
-    "lib/collins_shell/console/command_helpers.rb",
-    "lib/collins_shell/console/commands.rb",
-    "lib/collins_shell/console/commands/cat.rb",
-    "lib/collins_shell/console/commands/cd.rb",
-    "lib/collins_shell/console/commands/io.rb",
-    "lib/collins_shell/console/commands/iterators.rb",
-    "lib/collins_shell/console/commands/tail.rb",
-    "lib/collins_shell/console/commands/versions.rb",
-    "lib/collins_shell/console/filesystem.rb",
-    "lib/collins_shell/console/options_helpers.rb",
-    "lib/collins_shell/errors.rb",
-    "lib/collins_shell/ip_address.rb",
-    "lib/collins_shell/ipmi.rb",
-    "lib/collins_shell/monkeypatch.rb",
-    "lib/collins_shell/provision.rb",
-    "lib/collins_shell/state.rb",
-    "lib/collins_shell/tag.rb",
-    "lib/collins_shell/thor.rb",
-    "lib/collins_shell/util.rb",
-    "lib/collins_shell/util/asset_printer.rb",
-    "lib/collins_shell/util/asset_stache.rb",
-    "lib/collins_shell/util/log_printer.rb",
-    "lib/collins_shell/util/printer_util.rb"
+    '.pryrc',
+    'Gemfile',
+    'Gemfile.lock',
+    'README.md',
+    'Rakefile',
+    'VERSION',
+    'bin/collins-shell',
+    'collins_shell.gemspec',
+    'lib/collins_shell.rb',
+    'lib/collins_shell/asset.rb',
+    'lib/collins_shell/asset_type.rb',
+    'lib/collins_shell/cli.rb',
+    'lib/collins_shell/console.rb',
+    'lib/collins_shell/console/asset.rb',
+    'lib/collins_shell/console/cache.rb',
+    'lib/collins_shell/console/command_helpers.rb',
+    'lib/collins_shell/console/commands.rb',
+    'lib/collins_shell/console/commands/cat.rb',
+    'lib/collins_shell/console/commands/cd.rb',
+    'lib/collins_shell/console/commands/io.rb',
+    'lib/collins_shell/console/commands/iterators.rb',
+    'lib/collins_shell/console/commands/tail.rb',
+    'lib/collins_shell/console/commands/versions.rb',
+    'lib/collins_shell/console/filesystem.rb',
+    'lib/collins_shell/console/options_helpers.rb',
+    'lib/collins_shell/errors.rb',
+    'lib/collins_shell/ip_address.rb',
+    'lib/collins_shell/ipmi.rb',
+    'lib/collins_shell/monkeypatch.rb',
+    'lib/collins_shell/provision.rb',
+    'lib/collins_shell/state.rb',
+    'lib/collins_shell/tag.rb',
+    'lib/collins_shell/thor.rb',
+    'lib/collins_shell/util.rb',
+    'lib/collins_shell/util/asset_printer.rb',
+    'lib/collins_shell/util/asset_stache.rb',
+    'lib/collins_shell/util/log_printer.rb',
+    'lib/collins_shell/util/printer_util.rb'
   ]
-  s.homepage = "https://github.com/tumblr/collins/tree/master/support/ruby/collins-shell"
-  s.licenses = ["APL 2.0"]
-  s.require_paths = ["lib"]
-  s.rubygems_version = "1.8.23"
```

Comment:
Docs say that this should never be set by hand, so I deleted it
http://guides.rubygems.org/specification-reference/#rubygems_version